### PR TITLE
live: Reuse --record option to keep recorded data

### PR DESCRIPTION
**Pull request by honggyukim**
The live command is used to record the data in a temporary directory and
discards it after the execution.

Someone might want to see the replay output immediately, but also wants
to keep the recorded data to run another analysis command.

The current --record option is only used for script command, but can
also be used for live command for this purpose.
```
  $ rm -rf uftrace.data
  $ uftrace --record -F main t-abc
  # DURATION     TID     FUNCTION
              [109070] | main() {
              [109070] |   a() {
              [109070] |     b() {
              [109070] |       c() {
     0.933 us [109070] |         getpid();
     3.617 us [109070] |       } /* c */
     4.207 us [109070] |     } /* b */
     4.844 us [109070] |   } /* a */
     9.650 us [109070] | } /* main */

  $ uftrace replay
  # DURATION     TID     FUNCTION
              [109070] | main() {
              [109070] |   a() {
              [109070] |     b() {
              [109070] |       c() {
     0.933 us [109070] |         getpid();
     3.617 us [109070] |       } /* c */
     4.207 us [109070] |     } /* b */
     4.844 us [109070] |   } /* a */
     9.650 us [109070] | } /* main */
```
Closes: #1038

Signed-off-by: Honggyu Kim <honggyu.kp@gmail.com>

### CODE REVIEW

File: cmds/live.c [hunk 1]
```diff
@@ -154,16 +156,13 @@ int command_live(int argc, char *argv[], struct opts *opts)
...
-
-			pr_out("\n#\n# uftrace replay\n#\n");
 		}
-
-		pr_dbg("start live-replaying...\n");
-		ret2 = command_replay(argc, argv, opts);
+		else {
+			pr_dbg("start live-replaying...\n");
+			ret2 = command_replay(argc, argv, opts);
+		}
```
Comment (namhyung) [inline cmds/live.c]: This part does not belong to this change, please remove.

Comment (honggyukim) [reply cmds/live.c]: Right, it was a mistake. I just updated the commit. Thanks!
